packages/pagination/src/Pagination/utils.ts

```diff
@@ -0,0 +1,18 @@
+export const getCurrentRangeString = (
+  itemsPerPage: number,
+  currentPage: number,
+  numTotalItems?: number,
+) => {
+  return `${itemsPerPage * (currentPage - 1) + 1} - ${Math.min(
+    itemsPerPage * currentPage,
+    numTotalItems ?? itemsPerPage * currentPage,
+  )}`;
```

Comment:
Might want to double check this for when `itemsPerPage`, `currentPage` and `numTotalItems` don't make sense together

Reply:
This is just a util to get a formatted string, so the focus should probably be on not allowing incorrect values as props